docs/imageset-config-ref.yaml

```diff
@@ -9,19 +9,25 @@ mirror:
     architectures:
       - "s390x" # Architectures to mirror for the collection of release versions (defaults to amd64)
     channels:
-      - name: stable-4.9 # References latest stable release
+      - name: stable-4.9 # References the latest stable release
       - name: stable-4.7 # Annotation references min and max version. 
         minVersion: '4.6.13'
         maxVersion: '4.7.18'
-    graph: true # Include Cincinnati upgrade graph image in imageset
+    graph: true # Include Cincinnati upgrade graph image in imageset (defaults to false)
   operators:
-    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.8 # References entire catalog
-      full: true # full can be set to pull a full catalog and must be set to filter packages
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.12 # References entire catalog
+      full: false # full set to false pull the latest version for all package channels with no versions set (default to false)
       packages:
+        - name: elasticsearch-operator
+          channels:
+            - name: 'stable-v0' # Mirrors a single channel
         - name: rhacs-operator
-          minVersion: '3.67.0'
+          minVersion: '3.67.0' # Mirrors all version from the minimum version to the latest in the package
```

Comment:
Add clarity

> Mirrors all versions from the minimum version to the latest in the package, including all channels that contain any of those versions.

Consider addition of extra note - not here, but somewhere - that "minmum" and "latest" don't mean semver-style sorting, but rather relate to the graph constructed through 'replaces' and 'skips' in published versions